src/signals/settings/categories/components/StandardTextsField/StandardTextsField.tsx

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import type { FormEvent } from 'react'
+import { useCallback, useEffect, useState } from 'react'
+
+import { ChevronDown, ChevronUp } from '@amsterdam/asc-assets'
+import { useParams } from 'react-router-dom'
+
+import Select from 'components/Select'
+import type { StandardText } from 'types/api/standard-texts'
+
+import {
+  ButtonWrapper,
+  DefaultTextBody,
+  StyledButton,
+  TextWrapper,
+  Wrapper,
+} from './styled'
+import { useFetch } from '../../../../../hooks'
+import configuration from '../../../../../shared/services/configuration/configuration'
+import statusList from '../../../../incident-management/definitions/statusList'
+import { Direction } from '../../types'
+import { orderStandardTexts } from '../utils'
+
+type Props = {
+  name: string
+  onChange: (standardTexts: StandardText[]) => void
+}
+
+export const StandardTextsField = ({ name, onChange }: Props) => {
+  const params = useParams<{ categoryId: string }>()
+  const { get, data } = useFetch<{ results: StandardText[] }>()
+  const [selectedStatus, setSelectedStatus] = useState<string>(
+    statusList[0].value
+  )
+  const [orderedStandardTexts, setOrderedStandardTexts] = useState<
+    StandardText[]
+  >([])
+
+  useEffect(() => {
+    get(
+      `${configuration.STANDARD_TEXTS_ENDPOINT}?ordering=statusmessagecategory__position&category_id=10&category_id=${params.categoryId}`
+    )
+  }, [get, params.categoryId])
+
+  const updatePosition = useCallback(
+    (index: number, direction: Direction) => {
+      const orderedTexts = orderStandardTexts(
+        direction,
+        index,
+        orderedStandardTexts
+      )
+
+      if (orderedTexts) {
+        setOrderedStandardTexts([...orderedTexts])
+        onChange(orderedTexts)
+      }
+    },
+    [onChange, orderedStandardTexts]
+  )
+  useEffect(() => {
+    if (data?.results) {
+      const state = statusList.find(
+        (status) => status.value === selectedStatus
+      )?.key
+
+      const orderedStandardTexts = data.results.filter((result) => {
+        return result.state === state
+      })
+      setOrderedStandardTexts(orderedStandardTexts)
+    }
```

Comment:
It is not possible to assign StandardTexts to all statusses. Just this list `changeStatusOptionList`. Is defined in the same file.